firefox_media_tests/eme/test_eme_playback.py

```diff
@@ -0,0 +1,57 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from media_test_harness.testcase import MediaTestCase
+from marionette_driver import Wait
+from marionette_driver.errors import TimeoutException
+import time
```

Comment:
style nit: group standard-library, 3rd-party and app-specific imports separately, in that order (i.e. `time` should move up, `MediaTestCase` should move down)